build/Targets/Imports.targets

```diff
@@ -321,6 +321,14 @@
     <ProjectCapability Remove="DependenciesTree"/>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="$(ToolsetCompilerPackageName)" Version="$(ToolsetCompilerPackageVersion)" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(UseRoslynAnalyzers)' == 'true'">
+    <PackageReference Include="Microsoft.Net.RoslynDiagnostics" Version="$(RoslynDiagnosticsNugetPackageVersion)" />
+  </ItemGroup>
```

Comment:
Can we make this a single item group and put the conditional on the PackageReference?